crypto/rc4/asm/rc4-x86_64.pl

```diff
@@ -140,7 +145,9 @@
 .globl	RC4
 .type	RC4,\@function,4
 .align	16
-RC4:	or	$len,$len
+RC4:
```

Comment:
while you are already here, could you please move the .cfi_startproc from line 154 to here?
Thanks!

Reply:
I will submit a separate patch.